### DISCUSSION TITLE
Is there any plan for an updated version of Mezzanine?

### DISCUSSION THREAD

**Discussion by MelvinDRM** [stephenmcd/mezzanine#2068] [General]
Hello community:

Mezzanine is, for me, the best CMS built in Django due to its versatility and friendliness for devs. I was wondering if there will be an updated version. I imagine that if I try to use it with Django 5, it will be quite the challenge to upgrade it.

Regards.

**Comment by kenbolton** [comment]
When I do an install from the main branch of the mezzanine code [pip installs Django>=2.2](https://github.com/stephenmcd/mezzanine/blob/e719286e06b5a9e376896ab7e4c27410c80c1586/setup.cfg#L44C5-L44C18) as evidenced from the output of `python manage.py runserver`:

``` asciiart
              .....
          _d^^^^^^^^^b_
       .d''           ``b.
     .p'                `q.
    .d'                   `b.
   .d'                     `b.   * Mezzanine 9999dev0
   ::                       ::   * Django 5.0.1
  ::    M E Z Z A N I N E    ::  * Python 3.11.7
   ::                       ::   * SQLite 3.45.0
   `p.                     .q'   * Darwin 22.6.0
    `p.                   .q'
     `b.                 .d'
       `q..          ..p'
          ^q........p^
              ''''
```

There is a test that fails and if you plan to use the built-in image gallery – specifically the thumbnail resizing function – apply this [one line change](https://github.com/stephenmcd/mezzanine/pull/2067/files#diff-1f8479ad3acd546b09d03e41971cfffad50e1812cd93ae4013bb0e3676b72952L437-R442) to `mezzanine/core/templatetags/mezzanine_tags.py`:

``` python
437-        image = ImageOps.fit(image, to_size, Image.ANTIALIAS, 0, to_pos)
422+        image = ImageOps.fit(image, to_size, Image.LANCZOS, 0, to_pos)
```

Will a new release be cut? With enough critical changes on the main branch, it seems likely!

**Comment by molokov** [comment]
I'm already running Django 5.0 with my website using Mezzanine - everything seems fine, with the change Ken mentions above (which is an issue I raised). Feel free to use my Fork, if you like - I've fixed a few bugs, but also have an extra change in there for the thumbnail tag which I'm not certain is the correct solution to pull back to main (although it's a good enough workaround for me).

**Comment by MelvinDRM** [comment]
Thank you both @kenbolton and @molokov for your replies.

First of all, nice to see that this is something possible without having too much to fix.

@kenbolton I'll keep this in mind

@molokov thank you, I'll sure use it. Thinking about this fork, I'm wondering if you decided to upgrade the TinyMCE version.

In general, I was worried because I was proposing a client to remake his site and I was wondering of the "future-proofness" of beloved Mezzanine.

Since we are here, I have an issue that I would like to run you by:

In a current project, my initial take was to use Mezzanine as an app instead of having it on the virtualenv, just to have more direct control and view over certain changes I wanted to make.

Everything was fine BUT there was one thing failing: the multilingual tabs in the Django admin were not showing/functioning properly. It seems it wasn't possible to find the assets related to that feature.

I checked everything: files in the correct place, media in the admin classes had the correct paths to files, but there was no way (or more time for that matter) to keep digging.

My decision was to have Mezzanine live in the virtualenv (again) and use the extra fields setting and the migration modules setting so I can make certain changes that will create migrations but managing to keep them outside the env (depending on what's your standard regarding migrations in a repo, having migrations in the env is a pain because it doesn't favor multiple instances of the same project)

Has this issue with the multilingual tab ever happened to either of you before?

In any case, once again, thanks for the hope you just gave me (:
